Fall-2017-JS02/Challenge2/project.css

```diff
@@ -1,7 +1,14 @@
 div.greeting {
-  color: rgb(0, 0, 250);
+  color: rgb(0, 0, 250); /*making color*/
```

Comment:
I like that you added comments to most things. Here, it might be a better comment to say what color is that mysterious rgb(0, 0, 25);